app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -150,6 +150,9 @@ class Config {
 
   String get cqLabelName => 'CQ+1';
 
+  // Default number of commits to return for benchmark dashboard.
+  int get maxRecords => 50;
```

Comment:
How did we get this number?

Reply:
This number inherits from existing logic (from GO backend more earlier). Up to change if necessary.